src/lib/utility/utilityLibrary.h

```diff
@@ -23,8 +24,8 @@ std::function<Ret(Args...)> loadFunctionFromLibrary(
 	const FilePath& libraryPath, const std::string& functionName, std::string& errorString)
 {
 #ifdef _WIN32
-	const std::string libraryPathString = libraryPath.getBackslashedString();
-	HINSTANCE handle = LoadLibrary(libraryPathString.c_str());
+	auto libraryPathString = QDir::toNativeSeparators(QString::fromStdWString(libraryPath.wstr()));
```

Comment:
Please don't use "auto", as we try to keep our code base explicit so that it is easily readable, even when using tools that don't infer a type for "auto".